### PR TITLE
CADENZA-33493 [Embedding FA] Export Table and Indicator as JSON

### PR DESCRIPTION
**Pull request by anistor**
https://jira.disy.net/browse/CADENZA-33493

### CODE REVIEW

**Review by Tobias43 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -211,6 +211,14 @@ const text = await response.text();
 ...
 ```
 
+Fetch data from a workbook view in JSON format.
```
Comment (jkissel) [inline README.md:214]: We do not need to showcase every combination of method / options / value in these examples. For getting all the details, the apidoc is the better tool. That's what I tried to explain in the orange box here: https://conf.disy.net/pages/viewpage.action?spaceKey=EHB&title=Cadenza+JS#CadenzaJS-ExamplesintheREADME

So for me, another media type is not important enough to "justify" another two examples. What do you think?

File: sandbox.html [hunk 2]
```diff
@@ -281,13 +288,34 @@
   </div>
   <div>
     <label for="mediaType">Media type *</label>
-    <input name="mediaType" id="mediaType" value="text/csv" required>
+    <select name="mediaType" id="mediaType" required>
+      <option>text/csv</option>
+      <option>application/vnd.openxmlformats-officedocument.spreadsheetml.sheet</option>
+      <option>application/json</option>
+      <option>application/pdf</option>
+    </select>
```
Comment (jkissel) [inline sandbox.html:296]: :+1:

File: src/cadenza.js [hunk 3]
```diff
@@ -642,6 +644,7 @@ function validGeometryType(value) {
 const MediaType = /** @type {Record<string, MediaType>} */ {
   CSV: 'text/csv',
   EXCEL: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+  JSON: 'application/json',
```
Comment (jkissel) [inline src/cadenza.js:647]: Thx for maintaining the alphabetic order :+1:

File: README.md (hunk 1 above)
Comment (anistor) [reply README.md:214]: Agreed, one media type showcased with a code snippet is enough.

**Pull request merged**